site/guides/enhanced-apis/token.md

```diff
@@ -3,22 +3,22 @@ outline: deep
 head:
   - - meta
     - property: og:title
-      content: How to Fetch a Smart Account's ERC-20 Tokens
+      content: How to Fetch a smart account's ERC-20 Tokens
```

Comment:
keep Smart Account capital-case on the metatags and section headers in this doc

Reply:
Done!

Kept capital-case in the titles and section headers but not in descriptions as everything is lower-case in descriptions.